src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Validation/InviteOrganizationUserValidator.cs

```diff
@@ -60,5 +79,8 @@ private async Task<ValidationResult<InviteUserOrganizationValidationRequest>> Va
         {
             return new Invalid<InviteUserOrganizationValidationRequest>(new Error<InviteUserOrganizationValidationRequest>(ex.Message, request));
         }
+
+        int GetSecretManagerSeatAdjustment(int occupiedSeats, InviteOrganization organization, int invitesToSend) =>
+            organization.SmSeats - (occupiedSeats + invitesToSend) ?? 0;
```

Comment:
I still don't think this is right. For example:

- I have a total of 10 SM seats
- subtract 9 occupied (1 seat remaining)
- add 3 new users with SM access

... would be 4 seats to add. But I'm only adding 3 users and I have 1 seat free, it should be 2.

I think it should be: `Math.Abs(organization.SmSeats - occupiedSeats - invitesToSend ?? 0)`.

Please add (or update) a test for this.